### ISSUE TITLE
telnet without arguments buffers its output until `open` establishes a connection or program exits

### ISSUE THREAD

**Issue by l2dy** [holzschu/a-shell#752]
Executing `telnet` does not immediately show the `telnet>` prompt. Until I run `open some.host 80` or press Ctrl+D, no new text is shown besides the commands I typed.

Tested on iOS 17.4 and a-Shell mini 1.13.0.

**Comment by holzschu** [comment]
You're right. It works as expected if you say `telnet some.host 80`, however. I'll have a look.

**Comment by holzschu** [comment]
This issue has been fixed in the TestFlight versions. It will be in the 1.14.0 release.
